Packs/PrismaCloudCompute/Integrations/PaloAltoNetworks_PrismaCloudCompute/PaloAltoNetworks_PrismaCloudCompute.yml

```diff
@@ -2722,6 +2729,185 @@ script:
     - contextPath: PrismaCloudCompute.RuntimeContainerAuditEvents.provider
       description: The provider of the container.
       type: String
+  - name: prisma-cloud-archive-audit-incident
+    description: Acknowledges an incident and moves it to an archived state.
+    arguments:
+      - name: incident_id
+        description: Incident ID.
+        isArray: false
+        required: true
+      - auto: PREDEFINED
+        name: action
+        description: Action for the command. archive - incident will be archived, unarchive - incident will be unarchived.
+        predefined:
+          - "archive"
+          - "unarchive"
+        defaultValue: "archive"
+    outputs:
+      - contextPath: PrismaCloudCompute.RuntimeContainerAuditEvents.test
+        description: The operating system of the container.
+        type: String
+  - name: prisma-cloud-compute-runtime-host-audit-events-list
+    description: Retrieves the runtime host audit events.
+    arguments:
+      - name: clusters
+        description: A comma-separated list of cluster names.
+        isArray: true
+      - name: namespaces
+        description: A comma-separated list of namespace names.
+        isArray: true
+      - name: audit_id
+        description: A comma-separated list of audit event IDs.
+        isArray: true
+      - name: profile_id
+        description: A comma-separated list of runtime profile IDs.
+        isArray: true
+      - name: image_name
+        description: A comma-separated list of image names.
+        isArray: true
+      - name: container
+        description: A comma-separated list of container names.
+        isArray: true
+      - name: container_id
+        description: A comma-separated list of container IDs.
+        isArray: true
+      - name: type
+        description: A comma-separated list of audit event types.
+        isArray: true
+      - name: effect
+        description: A comma-separated list of audit event effects.
+        isArray: true
+      - name: user
+        description: A comma-separated list of users.
+        isArray: true
+      - name: os
+        description: A comma-separated list of operating systems.
+        isArray: true
+      - name: app
+        description: A comma-separated list of applications.
+        isArray: true
+      - name: hostname
+        description: A comma-separated list of hostnames.
+        isArray: true
+      - name: time
+        description: Time is used to filter by audit time.
+      - name: attack_type
+        description: AttackTypes is used to filter by runtime audit attack type.
+      - name: limit
+        description: The maximum number of container scan reports to return. Must be between 1-50.
```

Comment:
Didn't we say that we will implement pagination behind the scenes? Such the for limit = 200 we will run several API call while changing the offset?